packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1125,7 +1247,7 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     if (_selection == value)
       return;
     _selection = value;
-    _selectionRects = null;
+    _selectionPainter.highlightedRange = value;
```

Comment:
technically this is not needed, since the selection painter will repaint with the `RenderEditable`.